### ISSUE TITLE
Missing AsciiDoc macro for 'menu:'

### ISSUE THREAD

**Issue by miwie** [dpmb/dpmb#13]
Markdown like `menu:...` is not handled in any output format.
Missing AsciiDoc macro?
[EDIT] `key:` also not handled

**Comment by xtaran** [comment]
Yes. Those are known issues if `asciidoc` is used -- which we use by default. Those macros are recognized if `asciidoctor` is used. You can play around with it with `make doctor-html`. So far I only managed to get the HTML build more or less working with `asciidoctor`.

Initially they were publisher-specific macros of our previous publisher. I didn't want to completely delete them as we will use them at some point, so I only renamed them to the macro names used by `asciidoctor`.

**Comment by miwie** [comment]
Maybe this helps:

> Menu and key macros for AsciiDoc

https://gist.github.com/mojavelinux/4736691
At least I get the intended text for `menu:` and `key:`

**Comment by xtaran** [comment]
Yay! That looks very promising, especially because it seems written by one of the `asciidoctor` authors. :-)
